packages/mechanic-template/app/pages/Function.js

```diff
@@ -50,7 +50,7 @@ export const Function = ({ name, exports, children }) => {
         height: bounds.height - 100
       };
     }
-    iframe.current.contentWindow.run(name, vals, true);
+    randomSeed.current = iframe.current.contentWindow.run(name, vals, true);
```

Comment:
I would suggest, as you mention in your PR description, that we have a ref here called `lastRun` that stores the entire mechanic object. This makes it possible to do things like stats, etc. Then you would use `lastRun.values.randomSeed` to set the seed for the export.

Reply:
Done!

Reply:
Thanks!